rust/Cargo.lock

```diff
@@ -0,0 +1,21 @@
+[root]
+name = "suricata"
+version = "4.0.0-dev"
```

Comment:
I think we could use that in configure then, right? Haven't tried, but not expecting side effects. Unless it rejects the dash for some reason.